barbados/BCO/isotope-analyzer-wv.yaml

```
@@ -0,0 +1,27 @@
+plugins:
+  source:
+    - module: intake_xarray
+
+sources:
+  CRDS:
+    description: BCO CRDS Continuous In-Situ Water Vapor Isotopic Composition
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/BARBADOS/BCO/PICARRO/EUREC4A_BCO_PICARRO_WVISOS_L1.nc
```

```suggestion
      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/BARBADOS/BCO/vapour/EUREC4A_BCO_PICARRO_WVISOS_L3.nc
```
That is the official version (i.e., the one linked from https://doi.org/10.25326/245)

---

Hi @leonie-villiger, just wanted to check if you had a chance to look into this again 😬

---

- A new version of the BCO precipitation isotope product with corrected datatype of `flag` variable has been uploaded and is now accessible via OPeNDAP. 
- Franziska will take care of the unsupported time dimension in the BCO PICARRO dataset.